cfgov/v1/tests/test_password_policy.py

```diff
@@ -81,12 +81,12 @@ def test_cant_set_password_if_locked(self):
 
 class TestPasswordReusePolicy(TestWithUser):
     def test_can_set_new_password(self):
-        user = self.get_user('password1')
+        user = self.get_user(last_password='password1')
         password_policy.validate_password_history(user, 'password2')
 
     def test_cant_reuse_password(self):
         password = 'password'
-        user = self.get_user(password)
+        user = self.get_user(last_password=password)
         self.assertRaises(
```

Comment:
This is nitpicky, but could you update this so it's consistent with the test above - e.g. passing in the string in both cases or passing in a variable in both cases.